src/aliases.js

```diff
@@ -0,0 +1,32 @@
+import { promises as fs } from 'fs'
+import { version as processVersion } from 'process'
+
+import findUp from 'find-up'
+
+export const NODE_VERSION_ALIAS = '.'
+export const NODE_VERSION_FILES = ['.node-version', '.nvmrc', '.naverc']
+
+export const CURRENT_NODE_ALIAS = '_'
+
+export const resolveNodeVersionAlias = async ({ cwd } = {}) => {
```

Comment:
```suggestion
const resolveNodeVersionAlias = async ({ cwd }) => {
```

This function is not used outside of this file, so should not need `export`.

The default value `{}` should not be needed either.